DevTools/HackStudio/main.cpp

```diff
@@ -692,8 +695,8 @@ int main(int argc, char** argv)
 
     g_open_file = open_file;
 
-    if (!argument_absolute_path.is_empty() && !argument_absolute_path.ends_with(".hackstudio"))
-        open_file(argument_absolute_path);
+    if (!workspace.is_empty() && !workspace.ends_with(".hackstudio"))
+        open_file(workspace);
```

Comment:
Not happy with the terminology here (and above) - I made it specifically so HackStudio will open either a project (probably what you mean by "workspace") **or** a regular file.

```cpp
open_file(workspace);
```

Doesn't really make sense.

Reply:
Ah, I see. Changed it back to `argument_absolute_path`.